client/build/static/js/2.d0237a51.chunk.js

```diff
@@ -1,3 +0,0 @@
-/*! For license information please see 2.d0237a51.chunk.js.LICENSE.txt */
```

Comment:
What do/did these files have to do with algolia?

Reply:
Nothing. Made the mistake of compiling the web app off this branch I believe, it created a bunch of cached files that I accidentally added to the pr, then I removed them. Incidental garbage clean up basically.